apps/common/models/i-casework.js

```diff
@@ -2,17 +2,16 @@
 
 const Model = require('hof').model;
 const crypto = require('crypto');
-
 const config = require('../../../config');
 
 module.exports = class CaseworkModel extends Model {
-  constructor(attributes, options) {
-    super(attributes, options);
-    this.options.timeout = this.options.timeout || config.icasework.timeout;
+  url() {
+    return `${config.icasework.url}${config.icasework.createpath}?db=flcms`;
```

Comment:
same as above, I would avoid hardcoding the db name, refer to the config

also I would use `encodeURIComponent`

Reply:
👍🏾 I have implemented this now